modal/object.py

```diff
@@ -117,6 +121,17 @@ def _init_from_other(self, other: O):
         # Transient use case, see Dict, Queue, and SharedVolume
         self._init(other._rep, other._load, other._is_another_app, other._preload)
 
+    @classmethod
+    def _from_other(cls: Type[O], other: "_Object"):
```

Comment:
minor thing but you might want to add `-> O` to the signature

Reply:
actually why is this a classmethod? it probably makes more sense as an instnace method:

```python
def clone(self) -> O:
   ...